shared/js/background/events.es6.js

```diff
@@ -19,7 +19,7 @@ const tdsStorage = require('./storage/tds.es6')
 const browserWrapper = require('./wrapper.es6')
 const limitReferrerData = require('./events/referrer-trimming')
 const { dropTracking3pCookiesFromResponse, dropTracking3pCookiesFromRequest } = require('./events/3p-tracking-cookie-blocking')
-const startup = require('./startup.es6')
+const { startupReady } = require('./ready.es6')
```

Comment:
Can this module just export `startup: {ready()}`? It would simplify the PR a lot.